homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -118,12 +118,18 @@ def __init__(self, opendata: OpendataTransport, name: str) -> None:
         self._opendata = opendata
         self._attr_name = name
         self._remaining_time: timedelta | None = None
+        self._connected = False
 
     @property
     def native_value(self) -> str:
         """Return the state of the sensor."""
         return self._opendata.connections[0]["departure"]
 
+    @property
+    def available(self) -> bool:
+        """Return the available of the sensor."""
+        return self._connected
+
     @property
```

Comment:
```suggestion

    @property
    def native_value(self) -> str:
        """Return the state of the sensor."""
        return self._opendata.connections[0]["departure"]

    @property
```

Reply:
Ahh, ok, so we can just use the parents implementation, got it :)

Reply:
I assume the default value is `false`